ts/src/coinbaseinternational.ts

```diff
@@ -0,0 +1,1850 @@
+            'timestamp': undefined,
+            'datetime': undefined,
+            'currency': code,
+            'amount': amount,
+            'fromAccount': fromAccount,
+            'toAccount': toAccount,
+            'status': success ? 'ok' : 'failed',
+        };
+    }
+
+    async createOrder (symbol: string, type: OrderType, side: OrderSide, amount: number, price: number = undefined, params = {}) {
+        /**
+         * @method
+         * @name coinbaseinternational#createOrder
+         * @description create a trade order
+         * @see https://docs.cloud.coinbase.com/intx/reference/createorder
+         * @param {string} symbol unified symbol of the market to create an order in
+         * @param {string} type 'market' or 'limit'
+         * @param {string} side 'buy' or 'sell'
+         * @param {float} amount how much you want to trade in units of the base currency, quote currency for 'market' 'buy' orders
+         * @param {float} [price] the price to fulfill the order, in units of the quote currency, ignored in market orders
+         * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @param {float} [params.stopPrice] price to trigger stop orders
+         * @param {float} [params.triggerPrice] price to trigger stop orders
+         * @param {float} [params.stopLossPrice] price to trigger stop-loss orders
+         * @param {bool} [params.postOnly] true or false
+         * @param {string} [params.tif] 'GTC', 'IOC', 'GTD'
+         * @param {string} [params.expire_time] The expiration time required for orders with the time in force set to GTT. Must not go beyond 30 days of the current time. Uses ISO-8601 format (e.g., 2023-03-16T23:59:53Z)
+         * @param {string} [params.stp_mode] Possible values: [NONE, AGGRESSING, BOTH] Specifies the behavior for self match handling. None disables the functionality, new cancels the newest order, and both cancels both orders.
+         * @returns {object} an [order structure]{@link https://docs.ccxt.com/#/?id=order-structure}
+         */
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        let typeId = type.toUpperCase ();
+        const stopPrice = this.safeNumberN (params, [ 'triggerPrice', 'stopPrice', 'stop_price' ]);
+        const clientOrderIdprefix = this.safeString (this.options, 'clientOrderId', 'ccxt');
+        let clientOrderId = clientOrderIdprefix + this.uuid ();
+        clientOrderId = clientOrderId.slice (0, 18);
+        const request = {
+            'client_order_id': clientOrderId,
+            'side': side.toUpperCase (),
+            'instrument': market['id'],
+            'size': this.amountToPrecision (market['symbol'], amount),
+        };
+        if (stopPrice !== undefined) {
+            if (type === 'limit') {
+                typeId = 'STOP_LIMIT';
+            } else {
+                typeId = 'STOP';
+            }
+            request['stop_price'] = stopPrice;
```

Comment:
can you place a stop order? having some difficulties with it

Reply:
It was requiring a to set the timeInForce to GTC. I made it now default to GTC, to make it easier and should work with: 
`cj coinbaseinternational createStopOrder BTC/USDC market buy 0.01 60000 59000 --test`